charts/redpanda/configmap.tpl.go

```diff
@@ -62,3 +62,114 @@ func RedpandaAdditionalStartFlags(dot *helmette.Dot, smp, memory, reserveMemory
 
 	return append(flags, values.Statefulset.AdditionalRedpandaCmdFlags...)
 }
+
+// func RedpandaYAMLKafkaListeners(dot *helmette.Dot) []KafkaListener {
```

Comment:
Was exploring the existing listener configurations. Will likely have a function that converts the existing shape into this structure and we'll convert our templates to use the new structure.